### PR TITLE
added usage to README.md

### PR DESCRIPTION
**Pull request by apmattil**
added usage for the first timers (as I was) how to start the debugger.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,6 +136,7 @@ Start tests with verbose flag:
 ```
 
 ## Key bindings
+<a name="keys"></a>
```
Comment (lloiser) [inline README.md]: there is no need to add this element here, you can use `#key-bindings` above to link to this section

File: README.md [hunk 2]
```diff
@@ -14,6 +14,11 @@ Either `apm install go-debug` or search for `go-debug` in the settings.
 
 If this fails you can still do it manually by using this guide: https://github.com/derekparker/delve/tree/master/Documentation/installation
 
+## Usage
+Use F5 to lauch the debug window (the one on the right).
```
Comment (lloiser) [inline README.md]: s/lauch/launch

File: README.md [hunk 3]
```diff
@@ -14,6 +14,11 @@ Either `apm install go-debug` or search for `go-debug` in the settings.
...
+At selected line press F9 to set (or togle) break point.
```
Comment (lloiser) [inline README.md]: s/togle/toggle

**Comment by apmattil** [comment]
well english is not my native language .. thanks.
at least I learned to do pull request ;)

**Comment by apmattil** [comment]
I hope I got it right now.

File: README.md [hunk 4]
```diff
@@ -14,6 +14,11 @@ Either `apm install go-debug` or search for `go-debug` in the settings.
 
 If this fails you can still do it manually by using this guide: https://github.com/derekparker/delve/tree/master/Documentation/installation
 
+## Usage
+Use F5 to launch the debug window (the one on the right).
+At selected line press F9 to set (or toggle) break point.
+other [other function keys](#key-bindings).
```
Comment (lloiser) [inline README.md]: Just one more minor thing: you have repeated `other` here. Just remove the first one :)

**Comment by lloiser** [comment]
Thank you very much for your help :+1:

**Comment by apmattil** [comment]
you are wellcome.

**Pull request merged**